### DISCUSSION TITLE
Who can help translating?

### DISCUSSION THREAD

**Discussion by bauerj** [bauerj/paperless_app#35] [Show and tell]
Currently, the app is only available in English and German.

If you speak any other language and have time to translate ~80 words, this can be done easily on https://crowdin.com/project/paperless-app

Is your language not listed there? Just let me know here and I will add it to the list.

It would be awesome if the community could help other people use Paperless App!

**Comment by mvdkleijn** [comment]
Going there now. (Dutch by the way)

**Reply by bauerj** [reply]
Awesome, thanks!

**Reply by mvdkleijn** [reply]
> Awesome, thanks!

Done and you're welcome. :smile:

**Reply by bauerj** [reply]
The next version will included your translations and also a list of all the translators in the "About" section of the app.

**Reply by axlgit** [reply]
Done for Italian and French

**Reply by bauerj** [reply]
Thanks, it will be released as an update next week! 🇫🇷🇮🇹🍻

**Reply by bauerj** [reply]
Version 0.0.11 has just been submitted to the Play Store and will contain all the latest translations.

Apologies to @mvdkleijn for the long delay.

**Reply by mvdkleijn** [reply]
No worries 😄

**Reply by bauerj** [reply]
The latest version contains some new strings. It would be awesome if you could translate those as well 🙏
